examples/rust/robby_fischer/README.md

```diff
@@ -0,0 +1,317 @@
+        for rank in 0..8 {
+            if let Some(piece) = board.position[file][rank] {
+                if piece.role != Role::Duck {
+                    let piece_model_info = self.piece_meshes.get(&piece).unwrap();
+
+                    // Calls the log method defined below.
+                    piece_model_info.log(&rec, &format!("a8origin/pieces/{file}/{rank}"));
+                }
+            } else {
+                // To remove the bounding box/mesh from the square we moved the piece from.
+                // You could clear both in one call using rerun::Clear::recursive but that
+                // would clear the transformation logged to "a8origin/pieces/{file}/{rank}"
+                // which is why it isn't done here.
+                rec.log(
+                    format!("a8origin/pieces/{file}/{rank}/mesh"),
+                    &rerun::Clear::flat(),
+                ).unwrap();
+                rec.log(
+                    format!("a8origin/pieces/{file}/{rank}/bounding_box"),
+                    &rerun::Clear::flat(),
+                ).unwrap();
+            }
+        }
+    }
+}
+
+// …
+
+impl PieceModelInfo {
+    pub fn log(&self, rec: &rerun::RecordingStream, entity_path: &str) {
+        self.bounding_box
+            .log(rec, &format!("{entity_path}/bounding_box"));
+        rec.log(format!("{entity_path}/mesh"), &self.model).unwrap();
+    }
+}
+```
+
+### Image
+
+To see the image projection in the 3D space view we must log the cameras transformation and it's intrinsic parameters.
+```rust
+// Computes the transformation `camera_to_a8` that goes from camera coordinates to board coordinates using the fiducial markers located at the corners of the board.
+// …
+
+let (_scale, rotation, translation) = camera_to_a8.to_scale_rotation_translation();
+rec.log(
+    "a8origin/pinhole",
+    &rerun::Transform3D::from_translation_rotation(translation, rotation),
+).unwrap();
+rec.log(
+    "a8origin/pinhole",
+    &rerun::Pinhole::from_focal_length_and_resolution(
+        [color_param.row(0)[0], color_param.row(1)[1]],
+        [640.0, 480.0],
+    ),
+).unwrap();
+```
+Then we can log the image to this path and it will be shown in the 3D view.
+```rust
+rec.log(
+    "a8origin/pinhole/image",
+    &rerun::Image::try_from(color_img.clone()).unwrap(),
+).unwrap();
+```
+
+Logs the mask along with the detected pieces.
+```rust
+rec.log("images/mask", &rerun::Image::try_from(mask).unwrap())
+                .unwrap();
+rec.log(
+    "images/points",
+    &Points2D::new(square_mid_points)
+        .with_labels(self.count_avg.iter().map(|cnt| cnt.to_string()))
+        .with_radii(with_pieces.iter().map(|b| if *b { 10.0 } else { 2.0 }))
+        .with_colors(is_white.iter().map(|&w| if w { [220; 3] } else { [50; 3] })),
+).unwrap();
+```
+
+### Blueprint
+There isn't a Rust API for blueprints so instead we have to create and log the blueprint from python.
+This was done by creating a script called "blueprint.py"
```

Comment:
Should link to the issue so that this doesn't _entire_ get out of date once we solved it

```suggestion
As of writing, there isn't a Rust API for blueprints so instead we have to create and log the blueprint from python (see [#5521](https://github.com/rerun-io/rerun/issues/5521)).
This was done by creating a script called "blueprint.py"
```